src/main/kotlin/org/kamiblue/client/module/modules/render/FogColor.kt

```diff
@@ -0,0 +1,23 @@
+package org.kamiblue.client.module.modules.render
+
+import org.kamiblue.client.module.Category
+import org.kamiblue.client.module.Module
+import net.minecraftforge.client.event.EntityViewRenderEvent
+import org.kamiblue.client.util.color.ColorHolder
+import org.kamiblue.event.listener.listener
+
+internal object FogColor : Module(
+    name = "FogColor",
+    description = "Recolors render fog",
+    category = Category.RENDER
+) {
+    private val Color = setting("Color", ColorHolder(111, 166, 222, 255))
```

Comment:
```suggestion
    private val color = setting("Color", ColorHolder(111, 166, 222), false)
```

Just disable alpha.

Reply:
I modified it separately since you changed the value name, but not where the value was being used (Lines 18-20)